examples/label_studio_annotation/materializers/pillow_image_materializer.py

```diff
@@ -0,0 +1,53 @@
+import os
+import tempfile
+from typing import Dict, Type
+
+from PIL import Image
+
+from zenml.artifacts import DataArtifact
+from zenml.io import fileio
+from zenml.logger import get_logger
+from zenml.materializers.base_materializer import BaseMaterializer
+from zenml.utils import io_utils
+
+logger = get_logger(__name__)
+
+
+class PillowImageMaterializer(BaseMaterializer):
+    """Materializer for PIL.Image objects.
+
+    This materializer takes a dictionary of files and returns a dictionary of
+    PIL image objects.
+    """
+
+    ASSOCIATED_TYPES = (dict,)
+    ASSOCIATED_ARTIFACT_TYPES = (DataArtifact,)
+
+    def handle_input(self, data_type: Type[Dict]) -> Dict:
+        """Read from artifact store"""
+        super().handle_input(data_type)
+        temp_dir = tempfile.TemporaryDirectory()
+        io_utils.copy_dir(self.artifact.uri, temp_dir.name)
```

Comment:
Are we sure this is needed? If so, how come this isn't an issue with other materializers? Or is it?

Reply:
We use this / do this pattern in at least one of our other materializers.